retry

```diff
@@ -8,6 +8,7 @@
 #   -t, --times  - Number of times to retry the command (default: 3)
 #   -s, --sleep  - Number of seconds to sleep between retries (default: 0)
 #
+source ~/.toolbox/toolbox
```

Comment:
Why is this needed?

Reply:
For retry to work with other functions that are scoped, like `checkout`, it needs to source them. 
eg. if you try naive `retry checkout` it will not work since checkout is using namespaced functions like `checkout::shallow()` which is not available.